aiohttp/helpers.py

```diff
@@ -402,8 +408,13 @@ def log(self, message, environ, response, transport, time):
         :param float time: Time taken to serve the request.
         """
         try:
-            self.logger.info(self._log_format % self._format_line(
-                [message, environ, response, transport, time]))
+            fmt_info = self._format_line(
+                [message, environ, response, transport, time])
+
+            extra = {name: value for name, value in fmt_info}
+            self.logger.info(self._log_format %
+                             tuple([value for _, value in fmt_info]),
```

Comment:
Actually reading the docs for the new formatter it seems like the logging module in python 3.2+ supports the new '{}' style formatting, making this solution a win-win and removes any hesitations I had for this solution, yay!  I'll code something up shortly.